processes/local_development/seed_test_data.py

```diff
@@ -17,7 +17,7 @@ def __init__(self, *args):
             'uuid' : uuid4(),
             'date_created' : datetime.now(timezone.utc).replace(tzinfo=None),
             'date_modified' :datetime.now(timezone.utc).replace(tzinfo=None),
-            'frbr_status': 'to_do',
+            'frbr_status': 'complete',
             'cluster_status': False,
             "source": 'test data source',
             'authors': ['Ayan||true'],
```

Comment:
Ok I think the has_part is going to cause you some trouble with the source!

Instead of met it should also be TEST_SOURCE. 

Lastly, I recommend just setting the identifiers to an empty list for now.